### PR TITLE
add a checkbox to enable bizum demo, this way a less than 5€ checkout…

### PR DESCRIPTION
**Pull request by jcuervas**
… will be generated

Proposal is simple, if we need to configure more specific scenarios for different PMs there would be more things to do.

### CODE REVIEW

File: server/views/checkout.hbs [hunk 1]
```diff
@@ -210,6 +210,21 @@
...
+          name="bizumDemo"
+          class="form-check-input"
+          value="true"
+          id="bizumDemo"
+          {{#if details.bizumDemo}}checked{{/if}}
+          onchange="toggleBizumDemo({{ details.bizumDemo }})"
+        />
+        <label class="form-check-label" for="bizumDemo">
+          Generate a checkout less than 5€, this will allow Bizum to be used. Cofidis won't be available.
+          Notice that Bizum has to be enabled in your account.
```
Comment (jimmyn) [inline server/views/checkout.hbs]: No need for this notice as this is a demo that uses our account. You should say something like: 

```
In test environment Bizum only supports transactions below 5 EUR. Generate a checkout less than 5€ to test Bizum.
```

**Review by jimmyn — APPROVED** [review]
We should also hide bizum button for big orders, otherwise it will be confusing for the user

File: server/views/checkout.hbs (hunk 1 above)
Comment (jcuervas) [reply server/views/checkout.hbs]: Updated, now bizum only is rendered for checkouts less or equal to 5€. But there's an issue if the payment is redirected to hosted page as that page does not render in this project so i have no control of this rule there

Comment (jimmyn) [reply server/views/checkout.hbs]: @jcuervas thats ok

**Pull request merged**